### PR TITLE
Update Postgresql JDBC to 42.2.10

### PR DESCRIPTION
**Pull request by kokosing**
Update Postgresql JDBC to 42.2.10

### CODE REVIEW

**Comment by kokosing** [comment]
Relates to #5962

**Comment by kokosing** [comment]
Automation failures are unrelated: 
 - https://github.com/prestosql/presto/issues/4864
 - https://github.com/prestosql/presto/issues/5928

**Comment by findepi** [comment]
> Automation failures are unrelated:
> 
> * #4864

https://github.com/prestosql/presto/pull/5971

File: presto-postgresql/pom.xml [hunk 1]
```diff
@@ -85,6 +85,14 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <!-- TODO remove this after fixing https://github.com/prestosql/presto/issues/5718 -->
```
Comment (findepi) [inline presto-postgresql/pom.xml:89]: I think we should do that issue instead.,

**Review by findepi — APPROVED** [review]

File: presto-postgresql/pom.xml (hunk 1 above)
Comment (kokosing) [reply presto-postgresql/pom.xml:89]: I would prefer not. Guava could be a pandora box.

**Pull request merged**